org.quassel_irc.QuasselClient.json

```diff
@@ -15,7 +17,8 @@
         "--talk-name=org.freedesktop.Notifications",
         "--talk-name=org.kde.StatusNotifierWatcher",
         "--talk-name=org.freedesktop.DockManager",
-        "--talk-name=net.launchpad.DockManager"
+        "--talk-name=net.launchpad.DockManager",
+	"--env=LD_LIBRARY_PATH=/app/lib:/app/lib/x86_64-linux-gnu"
```

Comment:
```suggestion
        "--env=LD_LIBRARY_PATH=/app/lib:/app/lib/x86_64-linux-gnu"
```

Reply:
This path is arch specific so that isn't going to work out.

Reply:
> > This makes quassel grow from 13.9M to 175.2M! I don't think it's a nice solution...
>
> What did you expect? QtWebengine is a bundled version of Chromium.

I didn't know a base-app would just copy all the contents on this app :/

> As with everything flatpak it does get de-duplicated when multiple apps use it.

Also in this case? I guess OSTree will do a good job to keep objects unique, but this time the visible size change in the CLI makes it annoying, even if this could be shared with other apps.